py/client/pydeephaven/_console_service.py

```diff
@@ -42,28 +42,6 @@ def run_script(self, server_script: str) -> Any:
         except Exception as e:
             raise DHError("failed to execute a command in the console.") from e
 
-    def open_table(self, name: str) -> Table:
-        """Opens a table by name."""
-        self.start_console()
-
-        try:
-            result_id = self.session.make_ticket()
-            response = self._grpc_console_stub.FetchTable(
-                console_pb2.FetchTableRequest(console_id=self.console_id,
-                                              table_id=result_id,
-                                              table_name=name),
-                metadata=self.session.grpc_metadata)
-
-            if response.success:
-                return Table(self.session, ticket=response.result_id.ticket,
-                             schema_header=response.schema_header,
-                             size=response.size,
-                             is_static=response.is_static)
-            else:
-                raise DHError("error open a table: " + response.error_info)
-        except Exception as e:
-            raise DHError("failed to open a table.") from e
```

Comment:
I don't understand why this is being removed.  It seems reasonable for the client to ask the server for a table by name.

Reply:
That one method is not the public api method.
The public API method open_table is on session.py
That method is a method that Colin realized (1) is wrong and (2) is unused
So Jianfeng asked me to remove it.




[4:01](https://illumon.slack.com/archives/C053QMX2LBW/p1694030468643529)
The public API method open_table is on session.py
[4:01](https://illumon.slack.com/archives/C053QMX2LBW/p1694030498138679)
That method that is being removed in the PR latest version as of now is a method that Colin realized (1) is wrong and (2) is unused
[4:01](https://illumon.slack.com/archives/C053QMX2LBW/p1694030504057739)
So Jianfeng asked me to remove it.




[4:01](https://illumon.slack.com/archives/C053QMX2LBW/p1694030468643529)
The public API method open_table is on session.py
[4:01](https://illumon.slack.com/archives/C053QMX2LBW/p1694030498138679)
That method that is being removed in the PR latest version as of now is a method that Colin realized (1) is wrong and (2) is unused
[4:01](https://illumon.slack.com/archives/C053QMX2LBW/p1694030504057739)
So Jianfeng asked me to remove it.